client/routes/namespace/workflow-archival/constants.js

```diff
@@ -1,11 +1,11 @@
 export const ARCHIVAL_STATUS_LIST = [
-  { value: '-1', label: 'Closed' },
-  { value: '0', label: 'Completed' },
-  { value: '1', label: 'Failed' },
-  { value: '2', label: 'Cancelled' },
-  { value: '3', label: 'Terminated' },
-  { value: '4', label: 'Continued As New' },
-  { value: '5', label: 'Timed Out' },
+  { value: 'CLOSED', label: 'Closed' },
+  { value: 'COMPLETED', label: 'Completed' },
+  { value: 'FAILED', label: 'Failed' },
+  { value: 'CANCELED', label: 'Cancelled' },
+  { value: 'TERMINATED', label: 'Terminated' },
+  { value: 'CONTINUED_AS_NEW', label: 'Continued As New' },
+  { value: 'TIMED_OUT', label: 'Timed Out' },
```

Comment:
These looks like `WorkflowExecutionStatus`. Where do they come from? Also there is no `CLOSED` but there are `RUNNING`. Are you sure they must be UPPERCASE?

Reply:
> Where do they come from?

This is the execution status. These constants are used to show the options of the status filter. 
![image](https://user-images.githubusercontent.com/11838981/86305941-beab3a00-bbc7-11ea-8bad-7d92cfb05c22.png)
Similar thing happens on worklow executions page

> Are you sure they must be UPPERCASE?

The casing doesn't matter as the server always turns them lowercase. https://github.com/temporalio/temporal/blob/master/common/archiver/filestore/queryParser.go#L250

> Also there is no CLOSED but there are RUNNING
My understanding is that archived workflow history is not going to have RUNNING status. QueryParser that uses ExecutionStatus doesn't expect RUNNING status neither. As for the CLOSED status - it was a case before as you can see in the old code. I'm going to remove it completely